static/espeasy_default.css

```diff
@@ -124,6 +124,11 @@ h6 {
 	font-size: 10pt
 }
 
+pre,
+xmp {
```

Comment:
Why have `xmp` in css if it's no longer used in the code?

Reply:
See my code piece a few comments above...

Reply:
It was removed in the commit just before my comment (W3C says it's obsolete since HTML3, and `pre` should be used instead), but didn't quote it